edb/language_server/server.py

```diff
@@ -66,6 +76,36 @@ class DiagnosticsSet:
         dataclasses.field(default_factory=lambda: {})
     )
 
+    def append(
+        self,
+        doc: pygls.workspace.TextDocument,
+        diagnostic: lsp_types.Diagnostic,
+    ):
+        if doc not in self.by_doc:
+            self.by_doc[doc] = []
+        self.by_doc[doc].append(diagnostic)
+
+    def extend(
+        self,
+        doc: pygls.workspace.TextDocument,
+        diagnostics: Iterable[lsp_types.Diagnostic],
+    ):
+        if doc not in self.by_doc:
+            self.by_doc[doc] = []
+        self.by_doc[doc].extend(diagnostics)
+
+
+def _new_diagnostic_at_the_top_at_the_top(message: str) -> lsp_types.Diagnostic:
```

Comment:
Does this intentionally stutter? `at_the_top_at_the_top`?

Reply:
Lol, no it does not - it's a new python language server, which is much worse than pylance.